rust/noosphere-cli/src/native/workspace.rs

```diff
@@ -201,7 +201,7 @@ impl Workspace {
 
         let content = self.read_local_content(pattern, new_blocks).await?;
 
-        let sphere_fs = SphereFs::at(&sphere_did, &sphere_cid, None, db);
+        let sphere_fs = SphereFs::at(&sphere_did, &sphere_cid, &Author::anonymous(), db).await?;
```

Comment:
I don't quite understand the meaning for an anonymous (new key) author here, could you explain?

Reply:
When you write to a sphere, you need a DID to be used as the author of the change; this brings a need to always have a DID that corresponds to the author. Eventually, this `SphereFs` construct will let you bounce from one sphere to the next by address book, moving your credentials along with it. In some spheres, your credential will give you read/write, in others read-only. An "anonymous" key is guaranteed to always be read-only.

A cheaper alternative would be to design the author to have some dummy DID that resolves to nothing, but generating a key is fairly cheap to do and would spare us from weird cases where someone tries to resolve a dummy DID for some reason.